This fails to build with:
```
openusd> -- Configuring done (4.6s)
openusd> CMake Error at /nix/store/v6zrwwq9l9zdd3bdz6mqwvasc77mjkz8-python3.11-materialx-1.39.1/lib/cmake/MaterialX/MaterialXTargets.cmake:222 (set_target_properties):
openusd>   The link interface of target "MaterialXRender" contains:
openusd>
openusd>     OpenImageIO::OpenImageIO
openusd>
openusd>   but the target was not found.  Possible reasons include:
openusd>
openusd>     * There is a typo in the target name.
openusd>     * A find_package call is missing for an IMPORTED target.
openusd>     * An ALIAS target is missing.
openusd>
openusd> Call Stack (most recent call first):
openusd>   /nix/store/v6zrwwq9l9zdd3bdz6mqwvasc77mjkz8-python3.11-materialx-1.39.1/lib/cmake/MaterialX/MaterialXConfig.cmake:36 (include)
openusd>   cmake/defaults/Packages.cmake:315 (find_package)
openusd>   CMakeLists.txt:23 (include)
openusd>
openusd>
openusd> -- Generating done (0.9s)
openusd> CMake Warning:
openusd>   Manually-specified variables were not used by the project:
openusd>
openusd>     BUILD_TESTING
openusd>     CMAKE_INSTALL_BINDIR
openusd>     CMAKE_INSTALL_DOCDIR
openusd>     CMAKE_INSTALL_INCLUDEDIR
openusd>     CMAKE_INSTALL_INFODIR
openusd>     CMAKE_INSTALL_LIBDIR
openusd>     CMAKE_INSTALL_LIBEXECDIR
openusd>     CMAKE_INSTALL_LOCALEDIR
openusd>     CMAKE_INSTALL_MANDIR
openusd>     CMAKE_INSTALL_OLDINCLUDEDIR
openusd>     CMAKE_INSTALL_SBINDIR
openusd>     CMAKE_POLICY_DEFAULT_CMP0025
openusd>     PYSIDE_BIN_DIR
openusd>
openusd>
openusd> CMake Generate step failed.  Build files cannot be regenerated correctly.
```

--- a/pkgs/development/python-modules/materialx/default.nix
+++ b/pkgs/development/python-modules/materialx/default.nix
@@ -17,13 +17,13 @@
 
 buildPythonPackage rec {
   pname = "materialx";
-  version = "1.39.1";
+  version = "1.38.10";
 
   src = fetchFromGitHub {
     owner = "AcademySoftwareFoundation";
     repo = "MaterialX";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-WzzsY1hOWwJEqT/ZRLIoZDfKNvx1Yf6aFhA3ZcSPx+s=";
+    rev = "v${version}";
+    hash = "sha256-/kMHmW2dptZNtjuhE5s+jvPRIdtY+FRiVtMU+tiBgQo=";
   };
 
   format = "other";

--- a/pkgs/development/python-modules/openusd/default.nix
+++ b/pkgs/development/python-modules/openusd/default.nix
@@ -70,8 +70,9 @@ buildPythonPackage rec {
   patches = [
     (fetchpatch {
       name = "port-to-embree-4.patch";
-      url = "https://github.com/PixarAnimationStudios/OpenUSD/pull/2266/commits/4b6c23d459c602fdac5e0ebc9b7722cbd5475e86.patch";
-      hash = "sha256-yjqdGAVqfEsOX1W/tG6c+GgQLYya5U9xgUe/sNIuDbw=";
+      # https://github.com/PixarAnimationStudios/OpenUSD/pull/2266
+      url = "https://github.com/PixarAnimationStudios/OpenUSD/commit/c8fec1342e05dca98a1afd4ea93c7a5f0b41e25b.patch?full_index=1";
+      hash = "sha256-pK1TUwmVv9zsZkOypq25pl+FJDxJJvozUtVP9ystGtI=";
     })
   ];
 

--- a/pkgs/development/python-modules/openusd/default.nix
+++ b/pkgs/development/python-modules/openusd/default.nix
@@ -52,6 +52,7 @@ in
 buildPythonPackage rec {
   pname = "openusd";
   version = "24.08";
+  pyproject = false;
 
   src = fetchFromGitHub {
     owner = "PixarAnimationStudios";
@@ -64,8 +65,6 @@ buildPythonPackage rec {
     if python.stdenv.hostPlatform.isDarwin then darwin.apple_sdk_11_0.stdenv else python.stdenv;
 
   outputs = [ "out" ] ++ lib.optional withDocs "doc";
-
-  format = "other";
 
   patches = [
     (fetchpatch {

